assets/strings/en-us/strings.json

```diff
@@ -0,0 +1,79 @@
+{
+    "stringKey": "String Value",
+    "_stringKey.comment": "String comment",
```

Comment:
in this complete file add comments like this , 
where you have specified {0} in comment write what does this value indicate, this helps localization team have correct meaning of sentense

Reply:
Done